README.md

```diff
@@ -1 +1,42 @@
 # Terraform Provider for Kentik synthetic tests
+
+## Requirements
+
+- [Go](https://golang.org/doc/install) >= 1.16
+- [Terraform](https://www.terraform.io/downloads.html) >= 0.13
+
+## Install
+
+Build and install the plugin so that Terraform can find and use it:
```

Comment:
(nit) "... so that Terraform can use it." is sufficient, because if it cannot find it, it cannot use it either :).